### ISSUE TITLE
Are we planning to have a new release for this project?

### ISSUE THREAD

**Issue by afsalthaj** [dbrattli/OSlash#17]
I can see the latest release being 0.5.1 which is on April 2015.
I find this project super useful, and would like to get a newer release. Please let me know if I am missing something.

**Comment by dbrattli** [comment]
Yes, this project have not received any updates in a while. I've tried a few times but gotten stuck with `mypy` type checker problems. I'll give it another try ....

**Comment by dbrattli** [comment]
FYI: https://github.com/dbrattli/OSlash/pull/18/files

**Comment by dbrattli** [comment]
Please test: https://pypi.org/project/OSlash/0.6.2/

**Comment by afsalthaj** [comment]
Thanks @dbrattli .
I will test it out
